Avoids this error during builds for ARM:
aarch64-binfmt-P: Could not open '/system/bin/linker64': No such file or directory

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -12,7 +12,6 @@ requires = [
     # well as our build.rs for the rust/cffi bridge.
     "setuptools!=74.0.0,!=74.1.0,!=74.1.1,!=74.1.2",
 ]
-build-backend = "maturin"
 
 [project]
 name = "cryptography"
